Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -50,10 +50,11 @@ extension Issue {
     return self
   }
 
-  /// Record an issue when a running test fails unexpectedly.
+  /// Record an issue when a running test and an issue occurs.
```

Comment:
Please have @iamleeg review these documentation comments. :)

Reply:
There's definitely something weird here…"when a running test occurs"? I think this should be something like:
```suggestion
  /// Records an issue that a test encounters while it's running.
```